Makefile

```diff
@@ -8,9 +8,9 @@ FOREMAN_RHCI_TESTS_PATH=$(join $(FOREMAN_TESTS_PATH), rhci)
 FOREMAN_SMOKE_TESTS_PATH=$(join $(FOREMAN_TESTS_PATH), smoke)
 FOREMAN_TESTS_PATH=tests/foreman/
 FOREMAN_UI_TESTS_PATH=$(join $(FOREMAN_TESTS_PATH), ui)
-NOSETESTS=python -m cProfile -o $@.pstats $$(which nosetests)
-NOSETESTS_OPTS=-s --logging-filter=nailgun,robottelo --with-xunit\
-			   --xunit-file=foreman-results.xml
+PYTEST=python -m cProfile -o $@.pstats $$(which py.test)
+PYTEST_OPTS=-v --junit-xml=foreman-results.xml
```

Comment:
:+1:  for running with -v , we can quickly know if some test passed , we do not have to wait for all the tests to complete.

Also anything which shows progress of the automation, is good, not necessarily detailed logging is required.
We can anyways refer individual logs for it. :)